src/tests/roc_audio/test_depacketizer.cpp

```diff
@@ -39,6 +39,13 @@ const PcmFuncs& pcm_funcs = PCM_int16_2ch;
 } // namespace
 
 TEST_GROUP(depacketizer) {
+
+    channel_mask_t channels = {0x3};
+
+    SampleSpec sample_spec = SampleSpec(100, channels);
```

Comment:
Let's keep sample rate (100) and channel mask (0x3) global constants (anon enum above). This way it's easy to find all test parameters in one place.